app/src/main/java/edu/grinnell/kdic/MainActivity.java

```diff
@@ -40,7 +41,7 @@
 
```

Comment:
Document the class using Javadoc style 
See: https://www.tutorialspoint.com/java/java_documentation.htm